src/DiplomacyGame.js

```diff
@@ -0,0 +1,25 @@
+import { Game, TurnOrder } from 'boardgame.io/core'
+
+import { setupPlayers } from './rules/players'
+
+const DiplomacyGame = Game({
+  setup: (numPlayers) => ({
+    players: setupPlayers(numPlayers),
+    cells: Array(9).fill(null)
```

Comment:
Do we need to make sure numPlayers is <= 9?

(Should we set 9 as a const somewhere, you know, since this is overengineered?)

Reply:
Since there are rule sets for playing with less than 9 players, we'd definitely want this to be a configurable variable. A const for default number of players is definitely a way to start.